src/app/services/api.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { environment } from '@env/environment';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ApiService {
+
+  constructor(private http: HttpClient) { }
+
+  get(method: string, params?: any): Observable<any> {
+    return this.http.get<any>(`${environment.apiUrl}` + method)
```

Comment:
```suggestion
    return this.http.get<any>(`${environment.apiUrl}` + method, {params})
```
?

Reply:
I planned to finalize this method, as well as create the one for `post()`.
But in this particular extension we don't need `post()` yet.